netket/optimizer/qgt/qgt_jacobian_pytree.py

```diff
@@ -46,31 +67,82 @@ class QGTJacobianPyTree(LinearOperator):
     the field `sr`.
     """
 
-    O: jnp.ndarray = Uninitialized
-    """Gradients O_ij = ∂log ψ(σ_i)/∂p_j of the neural network 
-    for all samples σ_i at given values of the parameters p_j
-    Average <O_j> subtracted for each parameter
+    O: PyTree = Uninitialized
+    """Centred gradients ΔO_ij = O_ij - <O_j> of the neural network, where
```

Comment:
sorry, I love unicode
```suggestion
    """Centred gradients ΔO_ij = O_ij -  ⟨O_j⟩ of the neural network, where
```